src/components/SearchFiltersMinMax/stories/SearchFiltersMinMax.Default.stories.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Story from '../../../../stories/helpers/Story';
+
+import SearchFiltersMinMax from '../SearchFiltersMinMax';
+
+const STORY_COMPONENT = 'Search Filters Min/Max';
+const STORY_NAME = 'Default';
+
+const stories = storiesOf(`Components/${STORY_COMPONENT}`, module);
+
+stories.add('Default', () => {
```

Comment:
You can use the `STORY_NAME` constant here

```suggestion
stories.add(STORY_NAME, () => {
```